biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -407,9 +428,34 @@ real(r8) :: tcwood             ! Temperature response function for wood
                               
                               ! Scale for leaf nitrogen profile
                               nscaler = exp(-kn(ft) * cumulative_lai)
+                              
+                              ! Leaf maintenance respiration to match the base rate used in CN
+                              ! but with the new temperature functions for C3 and C4 plants.
+
+                              ! CN respiration has units:  g C / g N [leaf] / s. This needs to be
+                              ! converted from g C / g N [leaf] / s to umol CO2 / m**2 [leaf] / s
+                              
+                              ! Then scale this value at the top of the canopy for canopy depth
+                              ! Leaf nitrogen concentration at the top of the canopy (g N leaf / m**2 leaf)
+                              select case(hlm_parteh_mode)
+                              case (prt_carbon_allom_hyp)
+
+                                 lnc_top  = EDPftvarcon_inst%prt_nitr_stoich_p1(ft,leaf_organ)/slatop(ft)
+                                 
+                              case (prt_cnp_flex_allom_hyp)
+
+                                 leaf_c  = currentCohort%prt%GetState(leaf_organ, all_carbon_species)
+                                 leaf_n  = currentCohort%prt%GetState(leaf_organ, all_carbon_species)
+                                 lnc_top = leaf_n / (slatop(ft) * leaf_c )
+
+                              end select
+
+                              lmr25top = 2.525e-6_r8 * (1.5_r8 ** ((25._r8 - 20._r8)/10._r8))
```

Comment:
I realize that this is cut-and-pasted code, but I'm still slightly stressed that were not being consistent in our use of respiration functions through the code.  e.g. here we've got a q10 of 1.5, whereas in https://github.com/rgknox/fates/blob/rgknox-parteh/biogeophys/FatesPlantRespPhotosynthMod.F90#L1684 we're either using an arrhenius form or a q10 of 2 depending on whether we've got a c3 or c4 plant.  maybe this needs to be a separate issue?

Reply:
yeah, that sounds like a good new issue to create.  As you mentioned, this is just cut and paste, no methods were changed in the making of that code.

Reply:
ok, I'll make an issue